aten/src/ATen/native/tags.yaml

```diff
@@ -16,3 +16,8 @@
   desc: |
           This tag indicates that the operator doesn't have an explicit entry in
           native_functions.yaml, and instead was generated automatically by the codegen.
+- tag: nondeterministic
+  desc: |
+          This tag indicates that the operator is nondeterministic meaning that if the
+          operator is run twice with the same arguments, the outputs will not necessarily be
+          identical.
```

Comment:
we were discussing that we want to differentiate between:
a) numerical nondeterminism, for ops with implementations that don't guarantee bitwise equivalence across runs 
b) "actual" nondetermninism, for ops that intentionally produce different results, e.g. random ops.

and the intent of this tag was to only cover (b). Can you clarify this in the description?

cc @bdhirsh

Reply:
Naming bikeshed for the two categories above

I'm not sure what the best name for those two categories should be, but some ideas to bikeshed on:

(a) bitwise_nondeterministic
(b) nondeterministic

(a) nondeterministic
(b) from_distribution

(a) nondeterministic
(b) random

@albanD @anjali411 any thoughts / ideas / suggestions on names? Since once we pick one, we're stuck with it :)

Reply:
I think `seeded_nondeterministic` would be clearer (since the ops in question are not truly nondeterministic -- they generate the same output for a given seed). cc. @pbelevich who previously worked on RNG 

As for `bitwise_nondeterminism`, it sometimes only applies for an op on certain backends I think (cc. @ngimel). So, idk if we want to add a tag for that or not.

Reply:
I tentatively vote to swap the wording to `nondeterministic_seeded` and `nondeterminstic_bitwise`, so users are forced to see both and decide which one they want.